src/sentry/static/sentry/app/components/events/interfaces/threads/threadsSelectorOption.tsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import styled from '@emotion/styled';
+
+import InlineSvg from 'app/components/inlineSvg';
+import Badge from 'app/components/badge';
+import Flex from 'app/components/flex';
+import Text from 'app/components/text';
+import TextOverflow from 'app/components/textOverflow';
+import space from 'app/styles/space';
+import {Event} from 'app/types';
+
+import getThreadDetails, {ThreadDetails, Thread} from './getThreadDetails';
+import ThreadsSelectorOptionLabel, {
+  ThreadsSelectorOptionLabelType,
+} from './threadsSelectorOptionLabel';
+
+const StyledInlineSvgContainer = styled(Flex)(({theme}) => ({
+  color: theme.orange,
+  minHeight: 28,
+}));
+
+const StyledBadge = styled(Badge)({
+  marginRight: space(0.75),
+});
+
+const StyledThreadName = styled(Text)({
+  marginBottom: space(0.5),
+  fontWeight: 600,
+});
+
+const StyledInfo = styled(Flex)({
+  minWidth: 0,
+  marginRight: space(0.5),
+});
+
+const StyledInfoDetails = styled(Flex)({
+  maxWidth: '100%',
+});
+
+const StyledContainer = styled(Flex)({
+  maxWidth: '100%',
+});
+
+const StyledFileName = styled('span')({
+  color: '#2c58a8',
```

Comment:
I would love to, too, but Armin suggested I use this color, because it is already in use and indicates a filename. I was not sure if I should add it in the theme now 
![image](https://user-images.githubusercontent.com/29228205/73257766-a2dcdb80-41c4-11ea-95d3-eb75236143db.png)